config.go

```diff
@@ -600,6 +602,16 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Limit the max orphan count to a sane vlue.
+	if cfg.MaxOrphanTxs < 0 {
```

Comment:
Fair enough.  I prefer to use appropriate types in general (if something can't be negative, make it unsigned), but I suppose the nicer error message is worth it.